node-webservice/react-client/src/Components/GameplayPage.jsx

```diff
@@ -161,19 +145,57 @@ class GameplayPage extends Component {
         <div className='gameplay-page'>
           <h3>Playing game {this.props.match.params.gameid}</h3>
           <Grid onCellClick={this.handleCellClicked} parts={this.state.parts}></Grid>
+          <button onClick={this.handleSubmit}> Submit </button>
+          <RobotJSONObjectForm />
```

Comment:
I think the ui functionality is far along enough that we dont need the RobotJSONObjectForm anymore